src/com/amazon/ion/IonNumber.java

```diff
@@ -15,16 +15,45 @@
 
 package com.amazon.ion;
 
+import java.math.BigDecimal;
+
 /**
- * The IonNumber interface is a fore runner of a common base for the
- * ion numeric value types.  Currently only IonDecimal extends this
- * interface. In due course IonFloat and IonInt will be added to the
- * the family.
+ * Common functionality of Ion {@code int}, {@code decimal}, and {@code float}
```

Comment:
This is a nitpick, but it's common functionality of the IonInt, IonDecimal, and IonFloat types.

Reply:
Shoot, that's a good point. I'll keep that in mind and try to knock out some nitpicks later in the quarter.